bundles/org.openhab.binding.openweathermap/README.md

```diff
@@ -417,16 +416,15 @@ Number:Temperature localTomorrowEveningApparent "Evening apparent Temperature [%
 Number:Temperature localTomorrowNightApparent "Night apparent Temperature [%.1f %unit%]"  <temperature> { channel="openweathermap:onecall:api:local:forecastTomorrow#apparent-night" }
 Number:Pressure localTomorrowPressure  "Barometric pressure [%.1f %unit%]"  <pressure> { channel="openweathermap:onecall:api:local:forecastTomorrow#pressure" }
 Number:Dimensionless localTomorrowHumidity  "Atmospheric humidity [%d %unit%]"  <humidity> { channel="openweathermap:onecall:api:local:forecastTomorrow#humidity" }
-Number:Temperature localTomorrowDewpoint "Dew point [%.1f %unit%]"  <Temperature> { channea="openweathermap:onecall:api:local:forecastTomorrow#dew-point" }
+Number:Temperature localTomorrowDewpoint "Dew point [%.1f %unit%]"  <Temperature> { channel="openweathermap:onecall:api:local:forecastTomorrow#dew-point" }
 Number:Speed localTomorrowWindSpeed  "Wind speed [%.1f km/h]"   <wind> { channel="openweathermap:onecall:api:local:forecastTomorrow#wind-speed" }
 Number:Angle localTomorrowWindDirection "Wind direction [%d %unit%]" <wind> { channel="openweathermap:onecall:api:local:forecastTomorrow#wind-direction" }
 Number:Speed localTomorrowGustSpeed "Gust speed [%.1f km/h]" <wind> { channel="openweathermap:onecall:api:local:forecastTomorrow#gust-speed" }
-Number:Dimensionless localTomorrowPrecipProbability  "Precipitation probability [%.1f]"    { channel="openweathermap:onecall:api:local:forecastTomorrow#probability" }
```

Comment:
There is a small typo: Should be `precip-probability`.

Reply:
done